Composer/packages/lib/shared/src/constant.ts

```diff
@@ -19,6 +19,12 @@ export const FieldNames = {
   DefaultCase: 'default',
   Cases: 'cases',
 };
+
+export enum ProvisionType {
```

Comment:
It's a TS type but defined in `constant.ts`.

Reply:
removed